composer.lock

```diff
@@ -1318,16 +1318,16 @@
         },
         {
             "name": "psr/http-client",
-            "version": "1.0.2",
```

Comment:
can we keep the composer lock out of this PR?

Reply:
removed 👍